resolvers/index.js

```diff
@@ -22,6 +23,22 @@ export const resolvers = {
       const isValidPassword = bcrypt.compare(password, existedUser.password);
       if (!isValidPassword) throw new ApolloError("incorrect password");
 
+      const userHistory = await History.findOne({ userId: existedUser._id });
+      const date = new Date();
```

Comment:
please add toISOString() to date variable istead of line 31 and 37

Reply:
fixed please check